apple-codesign/src/notarization.rs

```diff
@@ -311,8 +315,33 @@ impl Notarizer {
                 .load(),
         );
 
-        let s3_client = aws_sdk_s3::Client::new(&config);
+        let proxy = {
+          let http_proxy = env::var_os("http_proxy");
+          let proxy_uri = http_proxy.unwrap().to_str().unwrap().parse::<Uri>().unwrap();
```

Comment:
This needs to be conditional otherwise we'll crash unless the `http_proxy` environment variable is defined.